mesheryctl/internal/cli/root/system/start.go

```diff
@@ -235,6 +235,10 @@ func start() error {
 					temp.Environment = append(temp.Environment, fmt.Sprintf("%s=%s", "MESHERY_SERVER_CALLBACK_URL", viper.GetString("MESHERY_SERVER_CALLBACK_URL")))
 				}
 
+				if currCtx.GetProvider() != "" {
+					temp.Environment = append(temp.Environment, fmt.Sprintf("%s=%s", "ENFORCED_PROVIDER", currCtx.GetProvider()))
```

Comment:
Should this environment variable be just PROVIDER in that case?
// @Revolyssup

Reply:
The proposed logic uses `provider` field only when there is a need to enforce a provider. And currently that enforcement is done through the environment variable `ENFORCED_VARIABLE`. So the logic here makes sense.

Reply:
@Revolyssup, The logic here is correct as per the current implementation of #6440 , however as per @leecalcote's earlier comment, the environment variable should also be changed to PROVIDER instead of ENFORCED_PROVIDER, isnt it ?
i.e. should all references to ENFORCED_PROVIDER change to just PROVIDER in #6440 ?

Reply:
Good eye. Thank you, @hexxdump

Reply:
Okay makes sense. The reasoning for naming it `ENFORCED_PROVIDER` instead of `PROVIDER` was to get rid of any ambiguity that may arise by the word. So that looking at the variable at first glance does communicate the intent properly.

Reply:
@Revolyssup that difference is neither here nor there from the user's perspective who has had no previously exposed user-facing ability to preselect a provider (outside of env vars being set in their container runtime). Even if this selection was exposed in a user-facing way previously, there's no need to differentiate: users identify a single remote provider, and in doing so, both pre-select and enforce use of a single provider for that deployment. It's all one-in-the-same concept.